src/TwigJs/Compiler/ForCompiler.php

```diff
@@ -70,51 +70,43 @@ public function compile(JsCompiler $compiler, Node $node)
 
         $compiler
             ->write("var $seqName = ")
-//             ->write("\$context['_seq'] = twig_ensure_traversable(")
             ->subcompile($node->getNode('seq'))
-//             ->raw(");\n")
             ->raw(";\n")
         ;
 
         if ($node->hasNode('else') && null !== $node->getNode('else')) {
             $compiler->write("var $iteratedName = false;\n");
         }
 
-        if ($node->getAttribute('with_loop')) {
-            $compiler
-                ->write("var $loopName = {\n")
-                ->indent()
-            ;
+        $compiler
```

Comment:
it's better to build an array and then just `json_encode` + pretty

Reply:
Using json_encode